ledger-service/http-json/BUILD.bazel

```diff
@@ -304,7 +304,10 @@ alias(
 [
     da_scala_library(
         name = "integration-tests-lib-{}".format(edition),
-        srcs = glob(["src/itlib/scala/**/*.scala"]),
+        srcs = glob(
+            ["src/itlib/scala/**/*.scala"],
+            exclude = ["src/itlib/scala/**/CE.scala"] if edition == "ee" else ["src/itlib/scala/**/EE.scala"],
```

Comment:
This follows the hack used for `failurelib` to pass information to the test which differs depending on whether we're running EE or not.